Trigger onPillRemove callback when a pill is removed in ComboBox (#5088)
jpmorganchase/salt-ds: .changeset/smart-brooms-juggle.md

---
"@salt-ds/core": patch
---

> Added the trigger to `onPillRemove` callback when a pill is clicked and removed in a `ComboBox`.

Reviewer: I think this is a "fixed"

Reviewer: ```suggestion
Fixed `onPillRemove` not being called when a pill is clicked and removed in a `ComboBox`.
```

Author: Done ✅
